tests/hazmat/primitives/test_dsa.py

```diff
@@ -511,6 +518,10 @@ def test_prehashed_digest_mismatch(self, backend):
             private_key.sign(digest, prehashed_alg)
 
 
+@pytest.mark.supported(
+    only_if=lambda backend: backend.dsa_supported(),
+    skip_message="Does not support DSA.",
+)
```

Comment:
This isn't necessary, none of these tests touch the backend.

Reply:
Fixed